tempto-core/src/main/java/com/teradata/tempto/sql/SqlContexts.java

```diff
@@ -41,6 +42,11 @@ public static ViewContextProvider createViewAs(String selectSql)
         return new ViewContextProvider(viewName, selectSql, defaultQueryExecutor());
     }
 
+    public static ViewContextProvider createViewAs(String viewName, String selectSql, QueryExecutor queryExecutor)
+    {
+        return new ViewContextProvider(viewName, selectSql, queryExecutor);
```

Comment:
Ok, I will updated the commit message. I want to use a custom QueryExecutor in which I am creating a custom connection with a different user (say `alice`) than the default `hdfs` user. This is needed for the access control checks tests (for granting/revoking privileges, we need more than a single user).